server/public/plugin/api.go

```diff
@@ -1306,6 +1306,12 @@ type API interface {
 	// @tag User
 	// Minimum server version: 9.8
 	UpdateUserRoles(userID, newRoles string) (*model.User, *model.AppError)
+
+	// GetPluginId returns the plugin's ID.
+	//
+	// @tag ID
```

Comment:
Let's group this next to the other plugin methods
```suggestion
	// @tag Plugin
```